app/models/microcosm.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: microcosms
+#
+#  id          :bigint(8)        not null, primary key
+#  name        :string           not null
+#  description :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  slug        :string           not null
+#  location    :string           not null
+#  lat         :decimal(, )      not null
+#  lon         :decimal(, )      not null
+#  min_lat     :integer          not null
+#  max_lat     :integer          not null
+#  min_lon     :integer          not null
+#  max_lon     :integer          not null
+#
+
+class Microcosm < ActiveRecord::Base
+  extend FriendlyId
+  friendly_id :name, :use => :slugged
+  self.ignored_columns = ["key"]
```

Comment:
This is, I think, a side effect of having multiple migrations, so can be removed after the migrations are combined.

Reply:
yes